google-cloud-storage/src/main/java/com/google/cloud/storage/Storage.java

```diff
@@ -2230,8 +2282,23 @@ public static Builder newBuilder() {
    * Acl acl = storage.updateAcl(bucketName, Acl.of(User.ofAllAuthenticatedUsers(), Role.OWNER));
    * }</pre>
    *
+   * <p>Example of updating a new ACL entry on a requester_pays bucket with a user_project option.
+   * <pre> {@code
+   * String bucketName = "my_unique_bucket";
+   * Acl acl = storage.updateAcl(bucketName, Acl.of(User.ofAllAuthenticatedUsers(), Role.OWNER),
+   *     BucketSourceOption.userProject("myProject"));
+   * }</pre>
+   *
+   * @param bucket name of the bucket where the updateAcl operation takes place
+   * @param acl ACL to update
+   * @param options extra parameters to apply to this operation
    * @throws StorageException upon failure
    */
+  Acl updateAcl(String bucket, Acl acl, BucketSourceOption... options);
+
+  /**
+   * @see Storage#updateAcl(String, Acl, BucketSourceOption...)
```

Comment:
Please remove `Storage` as it is redundant here

Reply:
Done.